integration-test/common/run_sweep_tests.py

```diff
@@ -0,0 +1,13 @@
+import pytest
```

Comment:
Why does this library need to exist for the sake of one function?

Reply:
Right, we don't need this. Deleted.